docs/running-nix.md

```diff
@@ -5,9 +5,10 @@ Argo Workflows has some basic support for Nix which is enough to get Argo Workfl
 Here are the steps to follow:
 
   1. Modify your hosts file and set up a Kubernetes cluster according to [Running Locally](running-locally.md). Don't worry about the other instructions.
-  1. Install [Nix](https://nixos.org/download.html).
-  1. Run `nix develop --extra-experimental-features nix-command --extra-experimental-features flakes ./dev/nix/ --impure` (you can add the extra features as a default in your `nix.conf` file).
-  1. Run `devenv up`.
+  2. Install [`Nix`](https://nixos.org/download.html).
+  3. Install [`direnv`](https://devenv.sh/getting-started/)
```

Comment:
Devenv, not direnv.

Using direnv to invoke devenv might be neat though

Reply:
Ah ofc, thanks for catching this.